Documentation/Blazorise.Docs/Pages/Docs/Extensions/SignaturePad/SignaturePadPage.razor

```diff
@@ -0,0 +1,154 @@
+    <DocsPageSectionContent Outlined FullWidth>
+        <SignaturePadMinWidthExample />
+    </DocsPageSectionContent>
+    <DocsPageSectionSource Code="SignaturePadMinWidthExample" />
+</DocsPageSection>
+
+<DocsPageSection>
+    <DocsPageSectionHeader Title="SignaturePad MaxWidth example">
+        Add parameter MaxWidth to your <Code>SignaturePad</Code> extension to set the maximum width of the signature stroke.
+    </DocsPageSectionHeader>
+    <DocsPageSectionContent Outlined FullWidth>
+        <SignaturePadMinWidthExample />
+    </DocsPageSectionContent>
+    <DocsPageSectionSource Code="SignaturePadMaxWidthExample" />
+</DocsPageSection>
+
+<DocsPageSection>
+    <DocsPageSectionHeader Title="SignaturePad MinDistance example">
+        Add parameter MinDistance to your <Code>SignaturePad</Code> extension to set the minimum distance between two points on the signature pad before a new point is added to the stroke.
+    </DocsPageSectionHeader>
+    <DocsPageSectionContent Outlined FullWidth>
+        <SignaturePadMinWidthExample />
+    </DocsPageSectionContent>
+    <DocsPageSectionSource Code="SignaturePadMinDistanceExample" />
+</DocsPageSection>
+
+<DocsPageSection>
+    <DocsPageSectionHeader Title="SignaturePad Throttle example">
+        Add parameter Throttle to your <Code>SignaturePad</Code> extension to set the delay time in milliseconds between each point capture in the signature drawing process.
+    </DocsPageSectionHeader>
+    <DocsPageSectionContent Outlined FullWidth>
+        <SignaturePadMinWidthExample />
+    </DocsPageSectionContent>
+    <DocsPageSectionSource Code="SignaturePadThrottleExample" />
+</DocsPageSection>
+
+<DocsPageSection>
+    <DocsPageSectionHeader Title="SignaturePad Velocity Filter Weight example">
+        Add parameter VelocityFilterWeight to your <Code>SignaturePad</Code> extension to set the smoothness of the signature pad by adjusting the line thickness based on the velocity of the drawing.
+    </DocsPageSectionHeader>
+    <DocsPageSectionContent Outlined FullWidth>
+        <SignaturePadMinWidthExample />
+    </DocsPageSectionContent>
+    <DocsPageSectionSource Code="SignaturePadVelocityFilterWeightExample" />
+</DocsPageSection>
+
+
+
+<DocsPageSubtitle>
+    API
+</DocsPageSubtitle>
+<Heading Size="HeadingSize.Is3">
+    Attributes
+</Heading>
+<DocsAttributes>
+    <DocsAttributesItem Name="BackgroundColor" Type="string" Default="white">
+        Sets the color of the signature pad's background.
+    </DocsAttributesItem>
+    <DocsAttributesItem Name="PenColor" Type="string" Default="#000000">
+        Sets the color of the pen used to draw the signature.
+    </DocsAttributesItem>
+    <DocsAttributesItem Name="Bind-Value" Type="object" Default="null">
+        Binds the value of the signature pad to a specified variable.
+    </DocsAttributesItem>
+    <DocsAttributesItem Name="DotSize" Type="int" Default="2">
+        Sets the size of the dots used in drawing the signature.
```

Comment:
Instead of 

> Sets the size of the dots used in drawing the signature.

Write it more like

> Defines the size of the dots used in drawing the signature.

Same for all other parameters.